code/datums/components/crafting/recipes/recipes_weapon_and_ammo.dm

```diff
@@ -522,6 +522,45 @@
 	category = CAT_WEAPONRY
 	subcategory = CAT_WEAPON
 
+/datum/crafting_recipe/gun/hobomat
+	name = "Heavy pipe rifle (.50)"
+	result = /obj/item/gun/ballistic/rifle/hobo/hobomat
+	reqs = list(/obj/item/ammo_casing/a50MG/improvised = 3,
+	/obj/item/stack/sheet/mineral/wood = 10,
+	/obj/item/stack/sheet/metal = 20,
+	/obj/item/stack/crafting/metalparts = 10,
+	/obj/item/stack/sheet/mineral/titanium = 1,
+	/obj/item/stack/crafting/goodparts = 2,
+	/obj/item/stack/rods = 3)
+	tools = list(TOOL_WORKBENCH)
+	time = 120
+	category = CAT_WEAPONRY
+	subcategory = CAT_WEAPON
+
+/datum/crafting_recipe/gun/knifegun
```

Comment:
I'm like 20% sure these recipe paths exist somewhere else, do they? If they do, one of them'll overwrite the other.

Reply:
They are not, these two (the knifegun and the knucklegun) aren't craftable anywhere, not even disabled.

They're only referenced in trash piles, other loot sources and the starter gun kit